bin/floydsauto

```diff
@@ -323,8 +324,7 @@ if __name__ == "__main__":
     output_files = glob.glob('*.tar.gz')
     output_directory = os.path.join(option.output_directory, option.site, camera, option.day_obs,
                                     'specproc')
-    if not os.path.exists(output_directory):
-        os.makedirs(output_directory)
+    Path.mkdir(output_directory, exist_ok=True)
```

Comment:
You may have intended to specify the `parents=True` keyword parameter, in order to duplicate the functionality of `os.makedirs()`.